presto-tests/src/main/java/com/facebook/presto/tests/statistics/MetricComparator.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tests.statistics;
+
+import com.facebook.presto.cost.PlanNodeCost;
+import com.facebook.presto.execution.QueryPlan;
+import com.facebook.presto.execution.StageInfo;
+import com.facebook.presto.spi.statistics.Estimate;
+import com.facebook.presto.sql.planner.plan.PlanNode;
+import com.facebook.presto.sql.planner.plan.PlanNodeId;
+import com.facebook.presto.sql.planner.planPrinter.PlanNodeStats;
```

Comment:
Maybe all the classes from `planPrinter` package, except for the printer itself, should go to a `execution.stats` packege?

Classes in question:
```
OperatorHashCollisionsStats
OperatorInputStats
PlanNodeStats
PlanNodeStatsSummarizer
```